.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Run Make Image Script
+      run: ./make_img.sh
+    - name: Github Release
+      uses: meeDamian/github-release@v1.0.0
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        gzip: false
+        files: >
+          *.zip
```

Comment:
Can be just:

```yml
files: *.zip
```

NOTE: Don't remember if globs(`*`) work.

Reply:
```files: *.zip``` doesn't seem to parse but ```files: "*.zip"``` does